site/docs/encoding/axis.md

```diff
@@ -46,7 +46,9 @@ Besides `axis` property of a field definition, the configuration object ([`confi
 
 {:#properties}
 
-To customize axis, an `axis` object in [an encoding channel's definition](encoding.html) can contain the following properties:
+To customize axis, you can specify an `axis` object in [an encoding channel's definition](encoding.html).  This section lists all properties of axes.
+
+You might also find this [interactive article](https://beta.observablehq.com/@jheer/a-guide-to-guides-axes-legends-in-vega), which provides interactive demo of Axes and Legends in the underlying Vega language, useful for understanding these properties.
```

Comment:
Maybe shorten. "You might also find this interactive article useful to understand these properties. It demonstrates Axes and Legends in the underlying Vega language"